src/site/_includes/macros/shows-card.njk

```diff
@@ -7,34 +7,19 @@
 {% set description = show.description | i18n(locale) %}
 {% set url = show.url %}
 {% set data = show.data %}
-{% set hero = data.hero or null %}
+{% set image = data.hero or null %}
 
-<div class="w-card" role="listitem">
-  <article class="w-card-base {{ 'w-card-base__with--cover' if hero }}">
-    {% if hero %}
-      <div class="w-card-base__cover">
-        {#
-          We intentionally hide the hero image from screen readers because it appears out of context
-          from the rest of the article content and may be confusing. Since the link for the article is duplicated
-          in the heading down below, this might be an OK tradeoff.
-        #}
-        <a href="{{url}}" class="w-card-base__link" tabindex="-1" aria-hidden="true">
-          {% Img src=hero, alt=title, width="354", height="240", params={fit: 'center', h: '240', w: '354'}, class="shows-card" %}
-        </a>
-      </div>
-    {% endif %}
-    <div class="w-card-base__blurb">
-      <a href="{{url}}" class="w-card-base__link">
-        <h2 class="{{ 'w-card-base__headline--with-image' if hero else 'w-card-base__headline' }}">{{title}}</h2>
-      </a>
-      <div class="w-card-base__desc">
-        {% if description %}
-          <a class="w-card-base__link" tabindex="-1" href="{{url}}">
-            <p class="w-card-base__subhead">{{description}}</p>
-          </a>
-        {% endif %}
-      </div>
-    </div>
-  </article>
-</div>
+<article class="card">
+  {% if image %}
+    <a href="{{ url }}" aria-hidden="true">
+      {% Img src=image, alt=title, width="570", height="330", class="card__hero", params={fit: 'center', h: '330', w: '570'} %}
+    </a>
+  {% endif %}
+  <div class="card__content flow">
```

Comment:
This seem almost identical with author card and tags card. Should we have one card, and just pass in an image type (avatar, thumbnail, none)?